pyhive/hive.py

```diff
@@ -12,6 +12,7 @@
 import datetime
 import re
 from decimal import Decimal
+from http.client import CannotSendHeader
```

Comment:
can we making this fully qualfied?

```suggestion
from pyhive.http.client import CannotSendHeader
```

Reply:
nvm this is actually a package called http